config/application.rb

```diff
@@ -20,5 +20,11 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+    config.middleware.insert_before 0, Rack::Cors do
```

Comment:
I'm not a CORS expert, but it seems this is duplicative of the set_cors_header. I think using a library is probably better than the simple implementation I made, but we should pick one way to do CORS and be consistent.

Reply:
I removed `set_cors_header` since the library is already doing it. The library also makes handling the `OPTIONS` routes easier.